modules/consignment-api/templates/ecs_assume_role_policy.json.tpl

```diff
@@ -0,0 +1,13 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "ecs-tasks.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
```

Comment:
Is the `Sid` field needed, as there is no value for it?

Reply:
No you're right, bad copy and paste job.